### PR TITLE
Feature/848 action names improvments

### PR DESCRIPTION
**Pull request by mkondratek**
~Please hold on with reviews - I see that the name for sync by merge from the context menu (but on the current branch) is wrong.~

Please review the other messages in the bundle (lets try to do it once  and for all).

Once we agree on the messages we can update the docs (readme, and scenario recordings if we decide to).
What do you think about that?

I would rather update only the text references (and leave the scenario recordings for another issue). 😆 

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Git Machete IntelliJ Plugin
+# Git Machete IntelliJ  Plugin
```
Comment (mkondratek) [inline README.md:1]: A changes to the scenario descriptions are needed; and new recordings probably 😕

Comment (PawelLipski) [reply README.md]: At least the former! although the moment for the latter has also probably come... given the recent change in icons

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java [hunk 2]
```diff
@@ -53,6 +54,17 @@ public List<SyncToParentStatus> getEligibleStatuses() {
   protected void onUpdate(AnActionEvent anActionEvent) {
     super.onUpdate(anActionEvent);
     syncToParentStatusDependentActionUpdate(anActionEvent);
+    val presentation = anActionEvent.getPresentation();
```
Comment (PawelLipski) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java:57]: Huh why this new piece of logic is needed? 🤔

Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java:57]: Was it missing **regardless** of this action name improvements?

Comment (mkondratek) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java]: n/a (I ve modified the wrong class, it was late xd)

File: README.md (hunk 1 above)
Comment (mkondratek) [reply README.md]: I will cover changes to README.md text in this one then. For the rest I have created: 
https://github.com/VirtusLab/git-machete-intellij-plugin/issues/850

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseMergeParentIntoBranchAction.java [hunk 3]
```diff
@@ -55,6 +56,17 @@ public List<SyncToParentStatus> getEligibleStatuses() {
   protected void onUpdate(AnActionEvent anActionEvent) {
     super.onUpdate(anActionEvent);
     syncToParentStatusDependentActionUpdate(anActionEvent);
+    val presentation = anActionEvent.getPresentation();
```
Comment (PawelLipski) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseMergeParentIntoBranchAction.java:59]: Okay, still the same point... tl;dr how was/would the behavior be wrong if it weren't for this change?

Comment (mkondratek) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseMergeParentIntoBranchAction.java:59]: from a context menu, you can pick the currently checked out branch ofc. In that case, the action name "Checkout and do smth" sounds silly and is inaccurate b/c the branch is already checked out

Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseMergeParentIntoBranchAction.java:59]: Awww okay 🤦🏻

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**